aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -187,6 +195,16 @@ pub fn deserialize(value_bytes: &[u8]) -> u128 {
     bcs::from_bytes(value_bytes).expect("unexpected deserialization error in aggregator")
 }
 
+// Helper for tests, #[cfg(test)] doesn't work for cross-crate.
```

Comment:
#[cfg(any(test, feature = "fuzzing"))] is how we do cross-crate gating

Reply:
can't get it to work :/